app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -113,6 +113,13 @@ def validate_name(self, name):
 
         return name
 
+    def validate_hypothesis(self, hypothesis):
+        if hypothesis.strip() == NimbusExperiment.HYPOTHESIS_DEFAULT.strip():
+            raise serializers.ValidationError(
+                "Please describe the hypothesis of your experiment."
```

Comment:
This description is probably fine, and I'm sure some basic training in experimenter would make this point moot, but worth noting that I wasn't entirely sure, with the default text in the field, that this error message actually meant "you must change these values or write your own" (because on initial glance it looked like there _was_ a hypothesis value). But like I said, this might be nothing.

Reply:
Yeah it's definitely a training thing, they have to know how to fill that in correctly and it's more than we can put in a help text.